store/sqlstore/upgrade.go

```diff
@@ -1403,6 +1403,10 @@ func upgradeDatabaseToVersion610(sqlStore *SqlStore) {
 		}
 	}
 
+	// TODO OAUTH not sure if needed
+	sqlStore.CreateColumnIfNotExistsNoDefault("OAuthApps", "Scopes", "Text", "VARCHAR(1024)")
+	sqlStore.CreateColumnIfNotExists("OAuthApps", "AppsFrameworkAppID", "VARCHAR(26)", "VARCHAR(26)", "")
+
```

Comment:
Two comments/questions:

- This probably needs to be moved to 6.2 migration. 
- What's the reason for not setting the default for `Scopes` but doing it for `AppsFrameworkAppID`?

Reply:
I want to keep nil on the existing register. The idea of that is that nil has a special meaning (legacy oauth apps), and want to preserve it.